packages/radix-ui-themes/src/components/kbd.props.ts

```diff
@@ -10,8 +10,13 @@ const kbdPropDefs = {
     default: undefined,
     responsive: true,
   },
+  asChild: {
+    type: 'boolean',
+    default: undefined,
+  },
```

Comment:
I think we should start adding asChild in the prop defs on every component here so that it shows up in the docs. I'll do a more holistic pass on asChild for other components too.

Reply:
To be clear, it does show in the docs, but we inject that prop def manually I believe.

Reply:
Gotcha, I just checked and we do it like this

<img width="614" alt="image" src="https://github.com/radix-ui/themes/assets/8441036/e021c89d-775f-4862-b4c6-167fb65ca756">

Thought we don't because there has been a bunch of bug reports about missing and/or not working `asChild`

Would you agree it should come from the prop defs?

Reply:
I think it would make sense to ultimately come from prop defs.